docs/conf.py

```diff
@@ -231,7 +231,7 @@
 # Whitelist pattern for tags (set to None to ignore all tags)
 smv_tag_whitelist = r'\b(3.22.0-scylla|3.21.0-scylla)\b'
```

Comment:
we need to define each version in this file ?

Reply:
That's correct, the repo ``sphinx-scylladb-theme`` comes with [notes](https://github.com/scylladb/sphinx-scylladb-theme#multiversion-support) to configure multiversion suppor . Another possibility would be to define a regex to build all the docs tagged with the suffix ``scylla``. Nevertheless,  this might be problematic if you want to exclude some specific version in the future.